src/location-picker/location-picker.component.scss

```diff
@@ -0,0 +1,32 @@
+.locationPickerContainer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.card-header {
+  background-color: var(--omrs-color-bg-medium-contrast);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  border-bottom: 1px solid var(--omrs-color-bg-low-contrast);
+  border-radius: 0.25rem 0.25rem 0 0;
+}
+
+.welcome-msg {
+  text-transform: capitalize;
+  margin: 1rem 0;
+}
+
+.location-card {
+  max-width: 60rem;
+}
+
+.center {
+  text-align: center;
+}
+
+.location-submit-btn {
+  width: 21.75rem;
```

Comment:
Should we maybe introduce a `max-width: 100%`, too? The value seems to be rather arbitrary as well.